mteb/leaderboard/app.py

```diff
@@ -23,13 +23,6 @@
 from mteb.leaderboard.figures import performance_size_plot, radar_chart
 from mteb.leaderboard.table import scores_to_tables
 
```

Comment:
If the changes were only the following, i.e. 1) change to ERROR and 2) specify model_meta logger, the same effect can be achieved. See printout below.

```suggestion

logging.getLogger("mteb.load_results.task_results").setLevel(
    logging.ERROR
)  # Warnings related to task split
logging.getLogger("mteb.model_meta").setLevel(
    logging.ERROR
)
logging.getLogger("mteb.models.overview").setLevel(
    logging.ERROR
)  # Warning related to model metadata (fetch_from_hf=False)
```

Here is the printout:
```
(mteb) isaac@macbook mteb % python -m mteb.leaderboard.app
<frozen runpy>:128: RuntimeWarning: 'mteb.leaderboard.app' found in sys.modules after import of package 'mteb.leaderboard', but prior to execution of 'mteb.leaderboard.app'; this may result in unpredictable behaviour
* Running on local URL:  http://127.0.0.1:7860
* Running on public URL: https://e0b4a9baf5be43a504.gradio.live

This share link expires in 72 hours. For free permanent hosting and GPU upgrades, run `gradio deploy` from the terminal in the working directory to deploy to Hugging Face Spaces (https://huggingface.co/spaces)
```

Reply:
Makes sense.